blocks/site-details/site-details.js

```diff
@@ -1211,17 +1230,24 @@ export default async function decorate(block) {
               editButton.innerText = 'Edit';
               tableRow.lastElementChild.prepend(editButton);
 
-              editButton.onclick = () => {
-                editButton.classList.add('loading');
-                fetch(`https://admin.hlx.page/status/headwire-self-service/${project.projectSlug}/main${item.path}?editUrl=auto`).then((res) => res.json()).then((statusData) => {
-                  if (statusData?.edit?.url) {
-                    window.open(statusData.edit.url, '_blank');
-                  }
-                }).catch(/* do nothing */)
-                  .finally(() => {
-                    editButton.classList.remove('loading');
-                  });
-              };
+              // TODO: change to link if we drop drive support
+              if (!darkAlleyVariation) {
+                editButton.onclick = () => {
+                  editButton.classList.add('loading');
+                  fetch(`https://admin.hlx.page/status/headwire-self-service/${project.projectSlug}/main${item.path}?editUrl=auto`).then((res) => res.json()).then((statusData) => {
```

Comment:
Can you use async await instead here ? Its easiee to read and its ok if its blocking since its happening after a user action

Reply:
changed it to use await